ui/apps/platform/src/Containers/NetworkGraph/hooks/useNetworkPolicySimulator.ts

```diff
@@ -87,22 +87,6 @@ function useNetworkPolicySimulator({ simulation, clusterId }: UseNetworkPolicySi
 
     function setNetworkPolicyModification(action: SetNetworkPolicyModificationAction): void {
         const { state, options } = action;
-        if (state === 'ACTIVE') {
-            setSimulator({
-                state: 'ACTIVE',
-                networkPolicies: [],
-                error: '',
-                isLoading: true,
-            });
-        } else {
-            setSimulator({
-                state,
-                modification: null,
-                error: '',
-                isLoading: true,
-            });
-        }
-
```

Comment:
i removed this because it seemed unnecessary since `setSimulator` forces a rerender and also is set again in the switch statement below -- this removed one unnecessary render in the `NetworkGraph` component

Reply:
fwiw it seems like there's no impact on the network policy simulator (it worked fine when i tested it out) but @sachaudh should probably double check

Reply:
Well, it was added for the `isLoading: true` part. When the simulation changes, we want to show a loading indicator until the data returns. That's why it renders twice. Let us keep it if it isn't hindering performance